src/GraphQL.Tests/Subscription/SubscriptionSchemaWithAutoGraphType.cs

```diff
@@ -0,0 +1,42 @@
+#nullable enable
+
+using System.Reactive.Linq;
+using GraphQL.Types;
+
+namespace GraphQL.Tests.Subscription;
+
+public class SubscriptionSchemaWithAutoGraphType : Schema
+{
+    public SubscriptionSchemaWithAutoGraphType()
+    {
+        Query = new AutoRegisteringObjectGraphType<QueryType>();
+        Mutation = new AutoRegisteringObjectGraphType<MutationType>();
+        Subscription = new AutoRegisteringObjectGraphType<SubscriptionType>();
+
+        RegisterTypeMapping(typeof(Message), typeof(AutoRegisteringObjectGraphType<Message>));
```

Comment:
Alternative syntax:

`this.AutoRegister<Message>()`

Reply:
done